packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.scss

```diff
@@ -1,3 +1,5 @@
+@import '~@talend/bootstrap-theme/src/theme/animation';
```

Comment:
Is it really necessary to import it ?

Reply:
Yep to have access to mixin I have to import it :)